pegasus/router.rb

```diff
@@ -161,6 +154,41 @@ def self.set_max_age(type, default)
         base = settings.configs[base][:base]
       end
     end
+
+    @actionview ||= begin
+      # Lazily require actionview_sinatra here, because it it turn will require
+      # ActionView::Base, which will in turn run the ActiveSupport load hooks for
+      # the class.
+      #
+      # This can cause some issues for environments that want to load both
+      # Pegasus and Dashboard, since if ActionView is loaded outside the context
+      # of Rails it won't load all functionality, and ActionView won't be
+      # re-initialized when it _does_ get loaded by Rails.
```

Comment:
Wait - does this cause issues for our default development mode where pegasus runs as middleware for dashboard-server?

Reply:
yep; it doesn't matter for the production use case